app/app.py

```diff
@@ -58,6 +58,8 @@
 
 CONVERSATIONAL_MEMORY_SIZE = 3
 
+READABILITY_SCORE_THRESHOLD = 55.0
```

Comment:
Should this be a streamlit slider or a fixed constant?

Reply:
A fixed constant. User probably shouldn't be able to change this. 55 is what we came up with for now looking the chart [here](https://en.wikipedia.org/wiki/Flesch%E2%80%93Kincaid_readability_tests).